kafka/ed-fi-kafka-connect-transforms/config/checkstyle/java.header

```diff
@@ -0,0 +1,15 @@
+/\*
+ \* Copyright 20(19|2[0-9]) Aiven Oy
```

Comment:
Checkstyle appears to be a linter, and this [inspects each file for a header](https://checkstyle.org/checks/header/header.html). I wonder if we should modify to also look for the Ed-Fi license header?